packages/engine-core/src/binary/Connection.ts

```diff
@@ -54,8 +54,7 @@ export class Connection {
     if (this._pool) return
 
     this._pool = new Pool(url, {
-      connections: 100,
-      pipelining: 10,
+      connections: 1000,
```

Comment:
After:
* We instantiated up to 1000 Clients
* With each socket sending up to 1 requests

Reply:
(discussed with Dom, not an issue)